### PR TITLE
[15.0][MIG] stock_quant_cost_info: Migration to 15.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
- Module renamed from stock_inventory_cost_info to stock_quant_cost_info
- Add changes to allow show the cost difference when changing qty from stock.quants

cc @Tecnativa TT36551

ping @sergio-teruel @stefan-tecnativa

### CODE REVIEW

File: stock_quant_cost_info/tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from . import test_stock_inventory_cost_info
```
Comment (sergio-teruel) [inline stock_quant_cost_info/tests/__init__.py]: Rename this file

**Review by sergio-teruel — APPROVED** [review]
Minor comment

**Comment by CarlosRoca13** [comment]
Change done @sergio-teruel

**Comment by sergio-teruel** [comment]
Thanks!!

**Review by stefan-tecnativa — APPROVED** [review]

**Pull request merged**